docs/pages/reference/terraform/resources/app.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: "teleport_app Resource - teleport"
+description: |-
+  
```

Comment:
Highlighting the description as something to fill in for SEO etc.

Reply:
Having per-resource descriptions is a `protoc-gen-terraform` issue, I'll look if I can modify the protoc plugin to add descriptions in the TF schema. In the meantime I'll add some generic description:

> This page describes the supported values of the teleport_app resource of the Teleport Terraform provider.